modules/xmpp/JingleSessionPC.js

```diff
@@ -1103,7 +1103,11 @@ JingleSessionPC.prototype.newJingleErrorHandler = function(request, failureCb) {
             error.source = request.tree();
         }
 
-        error.session = this;
+        // Commented to fix JSON.stringify(error) exception for
+        // circular dependancies when we print that error.
+        // FIXME: Maybe we can include
+        // part of the session object
```

Comment:
I don't see a reason why the FIXME should be split on two lines with so very few words per line. Did it hit a character limit?